espnet2/train/preprocessor.py

```diff
@@ -556,6 +556,7 @@ def __init__(
         speech_name: str = "speech",
         text_name: List[str] = ["text"],
         tokenizer_encode_conf: List[Dict] = [dict(), dict()],
+        not_available_symbol: str = None,
```

Comment:
Can you explain about it and embed the explanation in the source coed?

Reply:
I'll put the following explanation down below.
```
        # not_available_symbol is a special symbol as placeholder in the text. e.g.
        #      "utt_id <na>" an item in the text input
        # Then such samples will not have the corresponding text signals.
        # The resulting tensor would be processed as torch.longtensor([-1])
```